Tests/Cells/BaseBrickCellTests.swift

```diff
@@ -80,7 +80,37 @@ class BaseBrickCellTests: XCTestCase {
         XCTAssertEqual(cell?.topSeparatorLine.backgroundColor, .blueColor())
         XCTAssertNil(cell?.bottomSeparatorLine.superview)
     }
-
+    
+    func testSectionFixedHeight() {
+        brickView.registerBrickClass(LabelBrick.self)
+        
+        let section = BrickSection(height: .Fixed(size: 100), bricks: [
```

Comment:
Could we also add, that when the first section's height is "fill", the section is the same height as the collection view's height. That would be a great feature for the iPad PDP